web/sites/default/config/views.view.user_admin_people.yml

```diff
@@ -1793,6 +1793,52 @@ display:
             default_group: All
             default_group_multiple: {  }
             group_items: {  }
+        field_division_name_value:
+          id: field_division_name_value
+          table: user__field_division_name
+          field: field_division_name_value
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: string
+          operator: word
```

Comment:
I don't think the current operator "contains any word" is the best choice for mutli-word searches like "business solutions". I suggest changing it to "contains all words".

Reply:
Thanks! Setting updated to matching "all words".